main.py

```diff
@@ -1 +1,23 @@
 #Mad Libs Generator Project
+//Loop back to this point once code finishes
```

Comment:
lines 2, 5, 12, 22 are not properly commented out. try using "#" to comment instead of "//"